quickinfo/quickinfo-war/src/main/java/org/silverpeas/components/quickinfo/control/QuickInfoSessionController.java

```diff
@@ -96,16 +95,6 @@ public QuickInfoComponentSettings getInstanceSettings() {
   public NewsByStatus getQuickInfos() {
     NewsByStatus newsByStatus =
         getQuickInfoService().getAllNewsByStatus(getComponentId(), getUserId());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "newsByStatus". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1488)